opencv-kinfu-samples/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.10)
+add_executable(kinfu-example main.cpp)
+
+find_package(k4a 1.2.0 QUIET)
```

Comment:
may be use 1.3.0 which is the latest available release at this point

Reply:
Oh, okay. And also tested with 1.3.0 on my system.